@xen-orchestra/lite/src/locales/en.json

```diff
@@ -14,6 +14,7 @@
   "cancel": "Cancel",
   "change-power-state": "Change power state",
   "change-state": "Change state",
+  "confirm-delete": "You're about to delete ",
```

Comment:
Translation should not be done by concatenating pieces of text.

Reply:
We need to apply a specific style to the other part of it. If you have a better idea for that, I'll take it.

Reply:
could something like this work?

`You're about to delete { objects }`
```ts
$t("confirm-delete", objects: <span>{$t("n-vms", {n: selectedRefs.length})}</span>)
```

Reply:
Translation can be set like this:
```json
{
  "about-to-delete": "Vous êtes sur le point de supprimer {0}"
}
```

Then it is possible to use the `i18n-t` component:
```vue
<i18n-t keypath="about-to-delete" tag="div">
  <span class="accent">50 VM</span>
</i18n-t>
```